frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -136,6 +141,12 @@ const UpdateStatus: React.SFC<UpdateStatusProps> = ({cv}) => {
   </React.Fragment>;
 };
 
+// TODO:  origin has no channels, so channel selection will not apply
+// How to determine if origin?
+const CurrentChannel: React.SFC<CurrentChannelProps> = ({cv}) => <button className="btn btn-link co-m-modal-link" onClick={() => (clusterChannelModal({cv}))}>
+  {getCurrentClusterChannel(cv)}
```

Comment:
I'd probably just use

```suggestion
  {cv.spec.channel || '-'}
```